OpenRA.Mods.D2/Widgets/D2LineWidget.cs

```diff
@@ -0,0 +1,47 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The d2 mod Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Widgets;
+using OpenRA.Primitives;
+
+namespace OpenRA.Mods.D2.Widgets
+{
+	public class D2LineWidget : Widget
+	{
+		public int Size = 3;
+		public Func<Color> GetColor;
```

Comment:
Is there a special reason this is a `Func` and not just a `Color`?

Reply:
This code copypasted from some of common mod widgets, and Func<Color> here just like in original location